mlperf_common/callbacks/logging.py

```diff
@@ -214,40 +204,6 @@ def _end_train_block(self, train_batch_size: int) -> None:
         )
         self.train_block_started = False
 
-    def _log_training_throughput(
-        self,
-        trainer_step: int,
-        train_batch_size: int,
-    ) -> None:
-        delta_t = self.timer.get_delta()
-        delta_step = trainer_step - self.previous_step
-        throughput = (delta_step * train_batch_size) / delta_t
-        mllogger.event(
-            key="tracked_stats",
-            metadata={"step": self.train_current_block * train_batch_size},
-            value={
-                "throughput": throughput,
-                "train_step_time": delta_t / delta_step,
```

Comment:
we need `train_step_time`

Reply:
alright, please fix it then

Reply:
specifically:

*  `_log_validation_throughput()` is called in `on_validation_end()`
* `_log_training_throughput()` is called in `on_validation_start()`

If there's a callback for training block end it might be appropriate to move the call to `_log_training_throughput()` in that callback instead of in the `on_validation_start()` callback.  But I don't know Nemo, so don't know if such a callback exists.